cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -343,6 +414,26 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             secretRef: 'redis-pass'
           }
           {{- end}}
+          {{- if .EventHubs}}
+          {
+            name: 'AZURE_EVENT_HUBS_NAME'
+            value: eventHubNamespace.outputs.name
+          }
+          {
+            name: 'AZURE_EVENT_HUBS_HOST'
+            value: '${eventHubNamespace.outputs.name}.servicebus.windows.net'
+          }
+          {{- end}}
+          {{- if .ServiceBus}}
+          {
+            name: 'AZURE_SERVICE_BUS_NAME'
+            value: serviceBusNamespace.outputs.name
+          }
+          {
+            name: 'AZURE_SERVICE_BUS_HOST'
+            value: '${serviceBusNamespace.outputs.name}.servicebus.windows.net'
+          }
```

Comment:
Do we need to expose the resource id at the end of this bicep file?

Reply:
Yes indeed -- updated